llvm/lib/ProfileData/Coverage/CoverageMapping.cpp

```diff
@@ -925,26 +926,45 @@ LineCoverageIterator &LineCoverageIterator::operator++() {
   return *this;
 }
 
-static std::string getCoverageMapErrString(coveragemap_error Err) {
+static std::string getCoverageMapErrString(coveragemap_error Err,
+                                           const std::string &ErrMsg = "") {
+  std::string Msg;
+  raw_string_ostream OS(Msg);
+
   switch (Err) {
   case coveragemap_error::success:
-    return "Success";
+    OS << "Success";
```

Comment:
It seems that most message are untested?

Note, the diagnostic formatting follows Clang. We don't use capitalization: https://llvm.org/docs/CodingStandards.html#error-and-warning-messages

Reply:
The diagnostic formatting of llvm-cov has been updated to follow Clang in the commit https://github.com/llvm/llvm-project/pull/65264/commits/521abebc6c3a64f7c4060564300f9725a5a1a751.